libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/CreateClassloaderEntitlement.java

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+package org.elasticsearch.entitlement.runtime.policy;
+
+public class CreateClassloaderEntitlement implements Entitlement {
+    @ExternalEntitlement
```

Comment:
What is this annotation for? Does it differentiate entitlements that can appear in policy files versus those we just grant to the server?

Reply:
Thinking some more... if we instead had a list somewhere of "entitlements allowed in policy files" that would also allow us to scan those classes once at initialization, which would also solve the reflection perf problem I was mentioning.

Reply:
> which would also solve the reflection perf problem I was mentioning.

That, I think, would be a very nice thing to add in a follow-up!

Reply:
As agreed during the sync, this is now in the PR!